### PR TITLE
Add example file attachment parameters

### PR DESCRIPTION
**Pull request by brkr19**
Added example file attachment parameters to go with the code pull request to follow (5f9bd51)

### CODE REVIEW

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Nice first-time contribution @brkr19! 🏆

**Comment by brkr19** [comment]
The pull request for the code that would enable this functionality is held up waiting for a third-party dependency release, so the documentation change will have to wait as well.

https://github.com/home-assistant/home-assistant/pull/13748

**Comment by frenck** [comment]
@brkr19 Sorry about this, party my bad.
Please next time, keep the PR template as it was (and fill in the blanks) and choose the correct branch to merge into (it should have been `next` in this case). This will prevent misjudgments like this one.

../Frenck

P.S. You might need to re-open a new PR after my revert has been merged.

**Pull request merged**
